diff --git a/vialectrum b/vialectrum
index 6029abb..beec018 100755
--- a/vialectrum
+++ b/vialectrum
@@ -454,6 +454,8 @@ if __name__ == '__main__':
         if config_options.get('server'):
             config_options['auto_connect'] = False
 
+    config_options['portable'] = True
+
     if config_options.get('portable'):
         config_options['electrum_path'] = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'vialectrum_data')
 
